README.md

```diff
@@ -652,3 +652,16 @@ if err := r.Close(); err != nil {
     log.Fatal("failed to close reader:", err)
 }
 ```
+
+## Developer
+
+There are some flaky and non-runable tests. Execute them locally with
```

Comment:
Please rephrase:
```suggestion
Subtle behavior changes in later Kafka versions have caused some historical tests to break, if you are running against Kafka 2.3.1 or later, exporting the `KAFKA_SKIP_NETTEST=1` environment variables will skip those tests:
```

Reply:
I've combined this with some of the other suggestions.